gnucash/gnucash-cli.cpp

```diff
@@ -124,21 +126,45 @@ Gnucash::GnucashCli::start ([[maybe_unused]] int argc, [[maybe_unused]] char **a
 
     if (m_quotes_cmd)
     {
-        if (*m_quotes_cmd != "get")
+        if (*m_quotes_cmd == "info")
         {
-            std::cerr << bl::format (bl::translate("Unknown quotes command '{1}'")) % *m_quotes_cmd << "\n\n"
-            << *m_opt_desc_display.get();
-            return 1;
+            auto quotes = gnc_get_quotes_instance();
+            if (quotes.check())
```

Comment:
Something didn't work as expected there. When I skipped quotes.check() call it was as if it was never run. I wonder whether that is because the internal variable is made static and hence the check function runs at a time it can't complete successfully yet if it runs at all. To investigate further.

Reply:
Hmm, can't reproduce. So probably was due to another logic error I have fixed since then.

Reply:
Actually I can reproduce - sometimes. I suspect a race condition between the instantiation of the static quotes_cached and the execution of binreloc. If binreloc hasn't run yet, the code will expect gnc-fq-check in the hardcoded CMAKE_BIN_DIR, which is always CMAKE_INSTALL_PREFIX/bin.
That's wrong on macOS, Windows and when running from the build dir. In the latter case it may work by chance if you have also run the install target at some point. If you did change gnc-fq-check.in beforehand,  you may run the wrong version of that file as well.
So to avoid this, I have added some code to explicitly reinstantiate quotes_cached on first use. By that time binreloc has done its thing.